app/models/concerns/helpable.rb

```diff
@@ -65,18 +65,12 @@ def register_url
       # update minted column after first successful registration in handle system
 
       if [200, 201].include?(response.status)
-        success = true
-        if minted.blank?
-          success = update(minted: Time.zone.now, updated: Time.zone.now)
```

Comment:
This still needs to happen though. We have to update the minted/updated date

Reply:
I've reinstate this, but without the variable assignment (so rubocop doesn't get mad)